File: components/jpa-handling-session/src/main/java/com/sv/youapp/component/authorization/entities/jpa/RedirectUriEntity.java
package com.sv.youapp.component.authorization.entities.jpa;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name = "client_redirect_uri")
@Getter @Setter @NoArgsConstructor @AllArgsConstructor
public class RedirectUriEntity {

	@Id @GeneratedValue(strategy = GenerationType.IDENTITY)
	private Integer id;

	@Column(nullable = false, length = 255)
	private String redirectUri;
}

File: authorization-server/src/main/kotlin/com/sv/youapp/authorization/entities/RedirectUriEntity.kt
package com.sv.youapp.authorization.entities

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "client_redirect_uri")
class RedirectUriEntity(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long,
    @Column(nullable = false, length = 255)
    var redirectUri: String,
)
